tests/test_generate.py

```diff
@@ -11,10 +11,11 @@ def test_generate_int_makes_size_default():
     requested_types = ["int"]
     # assume the doubling experiment is at 100
     current_size = 100
+    level = 1
     # the default generator will return a tuple with 100 in it
     # expected_tuple = (generate.generate_int(current_size),)
     # generate the data for the requested_types and the current_size
-    generated_data = generate.generate_data(requested_types, current_size)
+    generated_data = generate.generate_data(requested_types, current_size, level)
```

Comment:
In this case, we wouldn't need to specify `level=1` or use the parameter `level` anymore